public/index.html

```diff
@@ -42,9 +42,11 @@
 	<script type="text/javascript">
         window.config = {
             domainName: "https://192.168.88.38",
-            mainColor: "#3596f4",
-            dangerColor: "#da534f",
-            contrastColor: "#000",
+			theme: {
```

Comment:
indentation issue?

Reply:
Solved - two palettes were added to window.config.